src/EventListener/InitializeSystemListener.php

```diff
@@ -86,27 +86,28 @@ public function onKernelRequest(GetResponseEvent $event)
     /**
      * Initializes the system upon console.command.
      */
-    public function onConsoleCommand()
+    public function onConsoleCommand(ConsoleCommandEvent $event)
     {
-        $this->setConstants('FE', 'console');
-
-        $this->boot();
+        if ($event->getCommand() instanceof ContaoFrameworkDependentInterface) {
+            $this->setConstants('FE', 'console');
+            $this->boot();
+        }
     }
 
     /**
-     * Returns the request scope.
+     * Returns the TL_MODE value for the container scope.
      *
-     * @param Request $request The request object
-     *
-     * @return string The request scope
+     * @return string The value for TL_MODE
      */
-    private function getScopeFromRequest(Request $request)
+    private function getScopeFromContainer()
     {
-        if ($request->attributes->has('_scope') && 'backend' === $request->attributes->get('_scope')) {
+        if ($this->container->isScopeActive('backend')) {
             return 'BE';
+        } elseif ($this->container->isScopeActive('frontend')) {
+            return 'FE';
+        } else {
```

Comment:
This `else` is not needed and will increase cyclomatic complexity with some tools.
Please rewrite as:
```php
if ($this->container->isScopeActive('backend')) {
    return 'BE';
} elseif ($this->container->isScopeActive('frontend')) {
    return 'FE';
}
return null;
```